Tasks/MSBuildV1/package.json

```diff
@@ -15,7 +15,7 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "azure-pipelines-task-lib": "^3.1.2",
-    "msbuildhelpers": "file:../../_build/Tasks/Common/msbuildhelpers-1.197.0.tgz"
+    "msbuildhelpers": "file:../../_build/Tasks/Common/msbuildhelpers-2.208.0.tgz"
```

Comment:
Can we use `msbuildHelpers-v3` from npm instead? - https://github.com/microsoft/azure-pipelines-tasks/tree/master/common-npm-packages/MSBuildHelpers-v3

Reply:
@KonstantinTyukalov, yes, we can, but in [another PR](https://github.com/microsoft/azure-pipelines-tasks/pull/16433).